third-party/libfabric/libfabric-src/.appveyor.ps1

```diff
@@ -0,0 +1,16 @@
+[cmdletbinding()] Param()
```

Comment:
I see a bunch of dotFiles here that I don't see when I download the 1.13.2 tarball. Did you copy the release tarball in or is there any chance you copied a git clone in?

Reply:
TBH had some git issues that I thought I resolved but I guess not. Let me try again to clean things up.